source/Shuffle.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2017 Akshay Agarwal <agarwal.akshay.akshay8@gmail.com>
+ * All rights reserved. Distributed under the terms of the MIT license.
+ */
+#ifndef SHUFFLE_H
+#define SHUFFLE_H
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+
+void Swap(int* a, int* b);
```

Comment:
random_shuffle has got deprecated in the latest revisions and shuffle won't work with gcc2.
This one makes use of random_shuffle, just tried in my test branch...
AkshayAgarwal007/Tipster/c7e9164